aws/createTemplate.sh

```diff
@@ -89,27 +94,36 @@ function options() {
   GENERATION_PROVIDER="${GENERATION_PROVIDER:-${GENERATION_PROVIDER_DEFAULT}}"
   GENERATION_FRAMEWORK="${GENERATION_FRAMEWORK:-${GENERATION_FRAMEWORK_DEFAULT}}"
   GENERATION_INPUT_SOURCE="${GENERATION_INPUT_SOURCE:-${GENERATION_INPUT_SOURCE_DEFAULT}}"
+  GENERATION_TESTING="${GENERATION_TESTING:-${GENERATION_TESTING_DEFAULT}}"
 
-  # Check level and deployment unit
-  ! isValidUnit "${LEVEL}" "${DEPLOYMENT_UNIT}" && fatal "Deployment unit/level not valid" && return 1
+  # Skip context generation when testing
+  if [[ "${GENERATION_TESTING}" == "false" ]]; then
```

Comment:
Will it work if we just check if `GENERATION_TESTCASE` has been passed here? `GENERATION_TESTING` is introduced but isn't allowed to be set it explicitly. Not a big deal, just an extra variable.

Reply:
Removed testcase and swapped it to enable on GENERATION_INPUT_SOURCE - that's more what its controlling anyway